API/NoDocs/Microsoft.Extensions.Telemetry.Metrics/MetricsExtensions.cs

```diff
@@ -0,0 +1,12 @@
+// Assembly 'Microsoft.Extensions.Telemetry.Abstractions'
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Telemetry.Metrics;
+
+public static class MetricsExtensions
+{
+    [Experimental("EXTEXP0003", UrlFormat = "https://aka.ms/dotnet-extensions-warnings/{0}")]
+    public static IServiceCollection RegisterMetrics(this IServiceCollection services);
```

Comment:
This registers Meter\<T> right? It should also be removed.

Reply:
Done.